### PR TITLE
Roleplay inventory

### PR DESCRIPTION
**Pull request by DutcherNL**
I keep going, but this is the end stop. No more new PRs after this for at least 2 weeks.

Oh yeah, I added a roleplay system branch on the system with an implementation of the new inventory system.

### CODE REVIEW

File: inventory/urls.py [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+from inventory.models import Item
+
+
+class CatalogueConverter:
+    regex = '[\w.-]+'
+
+    def to_python(self, value):
+        content_types = Item.get_item_contenttypes()
+        for item_type in content_types:
+            # item_type = ContentType.objects.get()
```
Comment (EricTRL) [inline inventory/urls.py:16]: These things always manage to sneak in :P

File: roleplaying/migrations/0001_squashed_initial.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import inventory.models
+import roleplaying.models
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('roleplaying', '0001_initial'), ('roleplaying', '0002_auto_20210803_0009'), ('roleplaying', '0003_roleplayingitem_digital_version_file_name'), ('roleplaying', '0004_roleplayingsystem_image'), ('roleplaying', '0005_auto_20210808_1347'), ('roleplaying', '0006_roleplayingsystem_dice'), ('roleplaying', '0007_auto_20210809_2250'), ('roleplaying', '0008_auto_20210809_2253'), ('roleplaying', '0009_auto_20210809_2327'), ('roleplaying', '0010_auto_20210809_2328'), ('roleplaying', '0011_auto_20210809_2330')]
```
Comment (EricTRL) [inline roleplaying/migrations/0001_squashed_initial.py:12]: This isn't really needed as these migrations aren't in production

**Review by EricTRL — CHANGES_REQUESTED** [review]
I like the changes! :D
However, I feel like navigation between roleplay systems, their items, its catalogue, and its edit pages is a bit finnicky. Sometimes there's a button to go somewhere, but none to go back. As a result, I ended up clicking back-and-forth through several pages before I got to the page where I wanted to go to.
The navigation seems a tad bit different from the boardgame-navigation due to the introduction of the "systems"-page, which is probably why I'm having trouble.

Other than that there are few, mostly minor, points of feedback or suggestions.

**Review by DutcherNL — COMMENTED** [review]
Reply

**Review by EricTRL — APPROVED** [review]

**Review by EricTRL — DISMISSED** [review]

**Review by EricTRL — APPROVED** [review]

**Pull request merged**